airbyte-integrations/connectors/destination-databricks/src/main/java/io/airbyte/integrations/destination/databricks/DatabricksDestination.java

```diff
@@ -25,93 +25,60 @@
 package io.airbyte.integrations.destination.databricks;
 
 import com.fasterxml.jackson.databind.JsonNode;
-import com.google.common.collect.ImmutableMap;
-import io.airbyte.commons.json.Jsons;
+import io.airbyte.db.Databases;
 import io.airbyte.db.jdbc.JdbcDatabase;
-import io.airbyte.integrations.base.Destination;
-import io.airbyte.integrations.base.IntegrationRunner;
-import io.airbyte.integrations.destination.jdbc.AbstractJdbcDestination;
-import io.airbyte.protocol.models.AirbyteConnectionStatus;
-import io.airbyte.protocol.models.AirbyteConnectionStatus.Status;
-import java.io.File;
-import java.net.MalformedURLException;
-import java.net.URL;
-import java.net.URLClassLoader;
-import java.util.Optional;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import io.airbyte.integrations.base.AirbyteMessageConsumer;
+import io.airbyte.integrations.destination.ExtendedNameTransformer;
+import io.airbyte.integrations.destination.jdbc.SqlOperations;
+import io.airbyte.integrations.destination.jdbc.copy.CopyConsumerFactory;
+import io.airbyte.integrations.destination.jdbc.copy.CopyDestination;
+import io.airbyte.integrations.destination.jdbc.copy.s3.S3Config;
+import io.airbyte.integrations.destination.jdbc.copy.s3.S3StreamCopier;
+import io.airbyte.protocol.models.AirbyteMessage;
+import io.airbyte.protocol.models.ConfiguredAirbyteCatalog;
+import java.util.function.Consumer;
 
-public class DatabricksDestination extends AbstractJdbcDestination implements Destination {
+public class DatabricksDestination extends CopyDestination {
 
-  private static final Logger LOGGER = LoggerFactory.getLogger(DatabricksDestination.class);
-
-  public static final String DRIVER_CLASS = "com.simba.spark.jdbc.Driver";
-
-  // TODO: this isn't working yet!
-  public static void getDriver() throws MalformedURLException, ClassNotFoundException {
-    File driverJar = new File("/Users/phlair/Downloads/SparkDriver/SparkJDBC42.jar");
-    URL jarUrl = new URL("jar", "", "file:" + driverJar.getAbsolutePath() + "!/");
-    URLClassLoader myLoader = new URLClassLoader(new URL[] { jarUrl } );
-    myLoader.loadClass(DRIVER_CLASS);
-  }
+  private static final String DRIVER_CLASS = "com.simba.spark.jdbc.Driver";
 
   @Override
-  public AirbyteConnectionStatus check(JsonNode config) {
```

Comment:
The `CopyDestination` can check or create the SQL tables by calling the abstract method. So we only need to check the persistence in this implementation.